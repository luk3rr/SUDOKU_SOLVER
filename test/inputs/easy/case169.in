800027001 061000000 000006700 090200500 102703000 603900002 000300000 200475006 000000940 
