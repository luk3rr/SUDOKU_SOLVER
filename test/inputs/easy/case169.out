Solving the following grid:
8 0 0 | 0 2 7 | 0 0 1 
0 6 1 | 0 0 0 | 0 0 0 
0 0 0 | 0 0 6 | 7 0 0 
------+-------+------
0 9 0 | 2 0 0 | 5 0 0 
1 0 2 | 7 0 3 | 0 0 0 
6 0 3 | 9 0 0 | 0 0 2 
------+-------+------
0 0 0 | 3 0 0 | 0 0 0 
2 0 0 | 4 7 5 | 0 0 6 
0 0 0 | 0 0 0 | 9 4 0 


Solution found :')

8 3 4 | 5 2 7 | 6 9 1 
7 6 1 | 8 3 9 | 2 5 4 
9 2 5 | 1 4 6 | 7 3 8 
------+-------+------
4 9 8 | 2 6 1 | 5 7 3 
1 5 2 | 7 8 3 | 4 6 9 
6 7 3 | 9 5 4 | 8 1 2 
------+-------+------
5 4 6 | 3 9 8 | 1 2 7 
2 1 9 | 4 7 5 | 3 8 6 
3 8 7 | 6 1 2 | 9 4 5 

Algorithm: BFS
Total time: 53 ms
Total expanded states: 8779
